cf-audit.py

```diff
@@ -24,39 +25,44 @@ def main():
 
     queries = []
     if args.after:
-        queries.append(f'q=timestamp>{args.after}')
+        queries.append(f'created_ats[gt]={args.after}')
     if args.before:
-        queries.append(f'q=timestamp<{args.before}')
-    initial_request = f'/v2/events?{"&".join(queries)}'
+        queries.append(f'created_ats[lt]={args.before}')
+    initial_request = f'/v3/audit_events?{"&".join(queries)}'
     logging.info('getting %s', initial_request)
     cf_out = subprocess.check_output(['cf', 'curl', initial_request], universal_newlines=True)
     cf_out = json.loads(cf_out)
     if args.user:
-        events = [event for event in cf_out['resources'] if event['entity']['actor_username'] == args.user]
+        events = [event for event in cf_out['resources'] if event['actor']['name'] == args.user]
     else:
         events = cf_out['resources']
-    next_url = cf_out['next_url']
-    while next_url is not None:
+    raw_next_url = cf_out['pagination']['next']
+    next_url_split = urlparse(raw_next_url['href'])
+    next_url = next_url_split.path+'?'+next_url_split.query
+    while raw_next_url is not None:
         logging.info('getting %s', next_url)
         cf_out = subprocess.check_output(['cf', 'curl', next_url], universal_newlines=True)
         cf_out = json.loads(cf_out)
         if args.user:
-            resources = [event for event in cf_out['resources'] if event['entity']['actor_username'] == args.user]
+            resources = [event for event in cf_out['resources'] if event['actor']['name'] == args.user]
         else:
             resources = cf_out['resources']
         events.extend(resources)
-        next_url = cf_out['next_url']
-    
-    print(json.dumps(events))
+        raw_next_url = cf_out['pagination']['next']
+        if raw_next_url is not None:
+            next_url_split = urlparse(raw_next_url['href'])
+            next_url = next_url_split.path+'?'+next_url_split.query
+        else:
+            print(json.dumps(events))
         
 
 def get_args():
     logging.basicConfig(level=logging.INFO)
     parser = argparse.ArgumentParser(
         description=description
     )
-    parser.add_argument('--after', help="find events after this timestamp (timestamp should be ISO8601)")
-    parser.add_argument('--before', help="find events before this timestamp (timestamp should be ISO8601)")
+    parser.add_argument('--after', help="find events after this timestamp (timestamp UTC formart YYYY-MM-DDThh:mm:ssZ)")
+    parser.add_argument('--before', help="find events before this timestamp (timestamp UTC formart YYYY-MM-DDThh:mm:ssZ)")
```

Comment:
Thanks for being a bit more explicit on the format here, that helps!  Just a slight spelling correction:

```suggestion
    parser.add_argument('--after', help="find events after this timestamp (timestamp UTC format YYYY-MM-DDThh:mm:ssZ)")
    parser.add_argument('--before', help="find events before this timestamp (timestamp UTC format YYYY-MM-DDThh:mm:ssZ)")
```